requirements.txt

```diff
@@ -1,2 +0,0 @@
-numpy
```

Comment:
you can not remove numpy as it is used inside setup.py with :  `include_dirs     = np.get_include(),`

Reply:
yes, I just recovered the requirements.txt file. I wanted to get rid of it, but I guess this is more convenient.